server/manager/src/test/java/org/apache/accumulo/manager/tableOps/bulkVer2/PrepBulkImportTest.java

```diff
@@ -79,14 +79,11 @@ List<KeyExtent> createExtents(Iterable<String> rowsIter) {
 
   Iterable<List<KeyExtent>> powerSet(KeyExtent... extents) {
     Set<Set<KeyExtent>> powerSet = Sets.powerSet(Set.of(extents));
-
-    return Iterables.transform(powerSet, set -> {
+    return () -> powerSet.stream().map(set -> {
       List<KeyExtent> list = new ArrayList<>(set);
-
       Collections.sort(list);
-
       return list;
-    });
+    }).iterator();
```

Comment:
Not sure if its worth it but this could return a stream instead of an Iterable and then be used with `powerSet().forEach`. 

EDIT: It could potentially also accept a Stream of Extents too if that would be better.

Reply:
I would love to use Iterable less often, and Streams more. We definitely should do that. But that's a much larger change, as those kinds of things cascade. This was a minimal change to modernize this and avoid the Guava Iterables class.